api/v1alpha1/envoyextensionypolicy_types.go

```diff
@@ -54,11 +54,19 @@ type EnvoyExtensionPolicySpec struct {
 	Wasm []Wasm `json:"wasm,omitempty"`
 
 	// ExtProc is an ordered list of external processing filters
-	// that should added to the envoy filter chain
+	// that should be added to the envoy filter chain
 	//
 	// +kubebuilder:validation:MaxItems=16
 	// +optional
 	ExtProc []ExtProc `json:"extProc,omitempty"`
+
+	// LuaFilters is an ordered list of Lua filters
+	// that should be added to the envoy filter chain
+	//
+	// +kubebuilder:validation:MaxItems=16
+	// +optional
+	// +notImplementedHide
+	LuaFilters []LuaFilter `json:"lua,omitempty"`
```

Comment:
Prefer to keep the same naming pattern with Wasm and ExtProc.
```suggestion
	Lua []Lua `json:"lua,omitempty"`
```